src/pages/Cart.js

```diff
@@ -1,5 +1,23 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import CartTable from '../components/CartTable';
 
 export const Cart = () => {
-  return <div>Cart</div>;
+  const [productos, setProductos] = useState([]);
+
+  useEffect(() => {
+    const getProductos = async () => {
+      try {
+        const productosRequest = await fetch('data/products.json');
+        const productosJson = await productosRequest.json();
+        setProductos(
+          productosJson.data.products.items.filter(({ price }) => price < 30)
+        );
+      } catch (err) {
+        console.log(err);
+      }
+    };
+    getProductos();
+  }, []);
+
```

Comment:
I think this is the same case, we could create a new custom hook and implement it here

Reply:
Same as above